types/redux-saga-tester/index.d.ts

```diff
@@ -1,11 +1,13 @@
 // Type definitions for redux-saga-tester 1.0
 // Project: https://github.com/wix/redux-saga-tester#readme
-// Definitions by: Ben Lorantfy <https://github.com/BenLorantfy>, Law Smith <https://github.com/lawsumisu>
+// Definitions by: Ben Lorantfy <https://github.com/BenLorantfy>
+//                 Law Smith <https://github.com/lawsumisu>
+//                 Björn Böing <https://github.com/BreakBB>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.4
+// Minimum TypeScript Version: 4.3
```

Comment:
what TS feature is needed here that warrants bumping the minimum version to 4.3?

Reply:
Tbh: I thought it might be a good idea to increase the minimum version. Sounds like it is not, so I can surely revert this change 👍🏻

Reply:
I think the minimum version should be the lowest possible to support the most amount of people

Reply:
Thanks for the hint. I reverted the change, since no upgrade was needed and will keep it in mind.